.github/workflows/tests-with-minimum-versions.yml

```diff
@@ -71,7 +71,7 @@ jobs:
         # Install dependencies with minimum versions.
         pip uninstall -y alembic cmaes packaging sqlalchemy plotly scikit-learn
         pip install alembic==1.5.0 cmaes==0.9.1 packaging==20.0 sqlalchemy==1.3.0 numpy==1.20.3 tqdm==4.27.0 colorlog==0.3 PyYAML==5.1
-        pip install plotly==4.9.0 scikit-learn==0.24.2  # optional extras
+        pip install plotly==5.0.0 scikit-learn==0.24.2  # optional extras
```

Comment:
should we check this with the other maintainers? also, should we not update `pyproject.toml` ?

Reply:
Updating `pyproject.toml` would prevent `plotly-4.9.0` users from using other visualization tools or accessing other optuna updates, so I don't think that's desirable.
For whether to depend on the newer version of plotly or reimplment the logic ourselves, I think we could have a discussion. What do you think, @toshihikoyanase?

Reply:
I had a discussion with @toshihikoyanase . We agreed that
* Updating the required dependency would be undesirable.
* Since this functionality is experimental, we could add a nice assertion message on `plotly` version and reimplement the logic if users really need it.

Reply:
I think we may want to memorize why plotly versions are inconsistent between `pyproject.toml` and this configuration. Someone may try to fix the inconsistency. How about leaving a comment on it?

Reply:
I added a comment.